server/routes/notificationRouter.ts

```diff
@@ -39,8 +40,8 @@ export default function notificationRouter(router: Router, services: Services):
   postWithValidator(
     '/notifications/pause',
     [
-      check('pauseValue', 'Enter a number').exists({ checkFalsy: true }).isInt(),
-      check('pauseValue', 'Enter a number above 0').isInt({ min: 1, max: 99 }),
+      check('pauseValue', 'Enter a number').exists({ checkFalsy: true }).bail().isInt(),
```

Comment:
we were getting the `Enter a number` twice so need `bail` to prevent the double error